src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -253,6 +253,35 @@ private function loadForm()
         $this->frm->addImage('image');
         $this->frm->addCheckbox('remove_image');
 
+        // page auth related fields
+        // check if profiles module is installed
+        if (BackendModel::isModuleInstalled('Profiles')) {
+            // check data for auth_required
+            $auth_required = false;
+            if (isset($this->record['data']['auth_required']) && $this->record['data']['auth_required']) {
+                $auth_required = true;
+            }
+            // add checkbox for auth_required
+            $this->frm->addCheckbox('auth_required', $auth_required);
+            // get all groups and parse them in key value pair
+            $groupItems = \Backend\Modules\Profiles\Engine\Model::getGroups();
+            if (!empty($groupItems)) {
+                $groups = array();
+                foreach ($groupItems as $key => $item) {
+                    $groups[] = array('label' => $item, 'value' => $key);
+                }
+                // set checked values
+                $checkValues = array();
```

Comment:
why `checkValues`? I think `checkedGroups` or `enabledGroups` or something like it would be a better name